bin/start.sh

```diff
@@ -61,6 +61,12 @@ echo "source /opt/rh/rh-ruby23/enable" > /home/git/.bashrc
 ## Map PV repositories
 ln -sf $GIT_REPO_ROOT/repositories /home/git/repositories
 
+# Creating a pipe so that gitlab-shell processes can write into something that does not increase in size
+mkfifo /home/git/gitlab-shell/gitlab-shell.log
```

Comment:
might be worth outputing some logging to say it's starting this.
does the output from gitlab have to be differenciated from the output of sshd? Probably not, just asking the question.
What happens if the "tail" command exits? maybe ```( tail -f /home/file.log ; echo "logging stopped unexpectedly?") &```